Glossary.html

```diff
@@ -1,24 +1,40 @@
 <!DOCTYPE html>
 <html>
-<ul>
-    <li><a href="index.html">Home</a></li>
-    <li><a href="Glossary.html">Glossary</a></li>
-    <li><a href="Links.html">Index</a></li>
-<h1>Acronym GLossary</h1>
-    <ul>
-    <h3>HTML</h3>
-    <blockquote>Hypertext Markup Language</blockquote>
-    <h3>HTTP</h3>
-    <blockquote>Hypertext Transfer Protocol</blockquote>
-    <h3>IP</h3>
-    <blockquote>Internet Protocol<blockquote></ul>
-    
-   <ul> <h3>TCP</h3>
-    <blockquote>Transmission Control Protocol<blockquote></ul>
-    <ul><h3>FTP</h3>
-    <blockquote>File Transfer Protocol<blockquote></ul>
-    <hr></hr>
-    <blockquote>You can make anything by writng
-    <br><b>&mdash; C.S. Lewis</b></br></blockquote>
-    <p>&copy; 2019 <a href="mailto:Alison.Nadjiwon@student.saultcollege.ca?subjext=Message from your CSD120 website"> Alison Nadjiwon</a></p>
+<head>
+    <title>Alison's CSD120 Website</title>
+    <meta charset="utf-8">
+</head>
+<body>
+
```

Comment:
<body><header>...</header><main>...</main><footer>...</footer></body> should be the overall structure of all the pages